### PR TITLE
[26357] 3rd Party firewall deployed as NVA in AVS private cloud or SDDC

### PR DESCRIPTION
**Pull request by khushal08**
# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## Testing Evidence

- *Replace this with any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate)*

## As part of this Pull Request I have

- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale-for-AVS/pulls)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale-for-AVS/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.

### CODE REVIEW

**Comment by prasad3017** [comment]
@khensler could you please review and approve this PR?

File: networking/nsx-firewall-patterns/README.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+
+1. Select "Tier-1 Gateways" blade on right. ![t1-static-route04](../nsx-firewall-patterns/assets/t1-static-route04.jpg)
+2. Click three dots on the left to edit the gateway.
+3. Expand "STATIC ROUTES". ![t1-static-route05](../nsx-firewall-patterns/assets/t1-static-route05.jpg)
+4. Click "ADD STATIC ROUTES". ![t1-static-route06](../nsx-firewall-patterns/assets/t1-static-route06.jpg)
+5. Create a default static route.
+   1. Enter "default01" as the name.
+   2. Enter "0.0.0.0/1" as the network.
+   3. Click on "Next Hops" to add one. ![t1-static-route06](../nsx-firewall-patterns/assets/t1-static-route07.jpg)
+6. Set Virtual NVA IP Address as the next hop and set the scope to the segments attached to this Tier-1 gateway. ![t1-static-route07](../nsx-firewall-patterns/assets/t1-static-route08.jpg)
```
Comment (khensler) [inline networking/nsx-firewall-patterns/README.md:376]: Is the Scope necessary?

Comment (prasad3017) [reply networking/nsx-firewall-patterns/README.md:376]: @khushal08  Kindly check comments from Kenyon?

Comment (khushal08) [reply networking/nsx-firewall-patterns/README.md:376]: @khensler Good point. It is optional. Better to remove it to avoid confusion.

Comment (khushal08) [reply networking/nsx-firewall-patterns/README.md]: @khensler @prasad3017  Updated. Any other feedback?

**Review by khensler — APPROVED** [review]

**Pull request merged**